### PR TITLE
Add get_container docker command

### PR DESCRIPTION
**Pull request by antevens**

### CODE REVIEW

File: docker_utils/get_container.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# Import the standard shell library
+# shellcheck source=../shtdlib.sh disable=SC1091
+source "$(dirname "${0}")/../shtdlib.sh" &> /dev/null || ../shtdlib.sh &> /dev/null || source ./shtdlib.sh &> /dev/null || source shtdlib.sh
+
+# A helper function to ensure containers exist locally before launch
+function get_container {
+    debug 10 "get_container called with: ${*}"
+    args=( "${@}" )
+    if [ "${#args[@]}" -lt 3 ] ; then
+        color_echo red "${0} needs at least three arguments, none were provided"
```
Comment (mroote) [inline docker_utils/get_container.sh]: Would be good too specify which arguments should be required in this output, or add some optargs to specify each argument.

Comment (antevens) [reply docker_utils/get_container.sh]: Done and done.

File: docker_utils/get_container.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+# Import the standard shell library
+# shellcheck source=../shtdlib.sh disable=SC1091
+source "$(dirname "${0}")/../shtdlib.sh" &> /dev/null || ../shtdlib.sh &> /dev/null || source ./shtdlib.sh &> /dev/null || source shtdlib.sh
+
+# A helper function to ensure containers exist locally before launch
+# Requires three arguments, repo, path and tag
+function get_container {
+    debug 10 "${0} called with: ${*}"
+    args=( "${@}" )
+    if [ "${#args[@]}" -lt 3 ] ; then
```
Comment (hrpatel) [inline docker_utils/get_container.sh]: Should we use `-eq` so the `elif` can be an `else`?

File: docker_utils/get_container.sh [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        color_echo red "${0} needs at least three arguments, repo, path and tag"
+        return 64
+    elif [ "${#args[@]}" -gt 1 ] ; then
```
Comment (hrpatel) [inline docker_utils/get_container.sh]: What if `path` or `tag` are left out? There is no checking for their existence.

**Pull request merged**